crates/relayer/src/keyring.rs

```diff
@@ -287,9 +287,10 @@ impl KeyRing<Ed25519KeyPair> {
     }
 }
 
+// Why is this not a method on `ChainConfig`?
```

Comment:
Is there any reason this isn't a method on `ChainConfig`, rather than a bare function defined in this other module?